_posts/hardeep_nahal/2017-04-25-RBioc.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title:  "Data visualization using R and Shiny"
+breadcrumb: true
+author: hardeep_nahal
+date: 2017-04-25
+categories: hardeep_nahal
+tags:
+    - Software Engineering Club
+    - R
+    - Bioconductor
+    - Shiny
+teaser:
+    info: Data visualization plays an essential role in high-throughput biology. It allows researchers to explore, understand and communicate data in a way descriptive statistics cannot compete with. 
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+## Introduction
+
+The ability to explore large-scale datasets frequently requires the use of intuitive and interactive data visualization tools to help extract knowledge and understand complex datasets. One of the most frequently used tools in bioinformatics is [R] - a powerful statistical programming language with strong data-mining and data visualization capabilities. It offers a set of inbuilt functions and libraries to analyze and present data, such as heatmaps, boxplots, and scatter and/or line plots. It can for example, generate a heatmap to represent genes exhibiting high or low expression in certain conditions. These graphs are usually static and 2 dimensional, so sharing results usually means exporting the static format. In this blog, I will discuss two R packages called [shiny] and [d3heatmap], which can take the typical two-dimensional graph a step futher and generate an interactive web application graph built around your R analysis. This allows a user to inspect specific values by hovering the mouse over the cell, as well as zooming to a particular region of the graph by dragging a rectangle around the relevant area. In addition to generating publication-quality graphics, users can also share the graphs by simply sending a URL. 
```

Comment:
"offers a set of inbuilt" => "offers a set of built-in"
https://english.stackexchange.com/questions/129360/is-inbuilt-a-word-is-it-alright-to-use-it-or-should-i-use-built-in